automatic/chromium/chromium.nuspec

```diff
@@ -15,7 +15,7 @@
 
 ## Notes
 
-- This package installs by default to the `%programfiles(x86)%` instead of `%localappdata%\Chromium`.
+- This package installs the Chromium snapshots builds by default to the `%programfiles(x86)%` instead of `%localappdata%\Chromium`.
```

Comment:
Looks like you accidentally this little tweak to the nuspec

Reply:
ops, I'll go through my commits and figure out where I made that mistake

Reply:
Ah, I see what happened.
I always test the updating process before I push out a new commit, and since the `Readme.md` hadn't been updated (still contains the old message), it got replaced in the nuspec file.